.ci/jenkins/scripts/minikube.groovy

```diff
@@ -1,4 +1,4 @@
-minikubeVersion = env.MINIKUBE_VERSION ?: '1.28.0'
+minikubeVersion = env.MINIKUBE_VERSION ?: '1.30.1'
 minikubeKubernetesVersion = env.KUBERNETES_VERSION ?: '1.23.14'
```

Comment:
Don't we need to update also this to version 1.26.3 ?

Reply:
oh, thought I did ...

```suggestion
minikubeKubernetesVersion = env.KUBERNETES_VERSION ?: '1.26.3'
```